ilastik/applets/dataSelection/precomputedVolumeBrowser.py

```diff
@@ -85,29 +99,47 @@ def setup_ui(self):
         main_layout.addWidget(self.qbuttons)
         self.setLayout(main_layout)
 
+    def resetVolume(self):
+        self.qbuttons.button(QDialogButtonBox.Ok).setEnabled(False)
+        self.debug_text.setText("")
+        self.scale_combo.clear()
+        self.base_url = None
+
+    @property
+    def selected_url(self) -> Optional[str]:
+        if not self.base_url:
+            return None
+        url = urljoin(self.base_url + "/", self.scale_combo.currentText())
```

Comment:
I actually need the url to have both protocols: `precomputed://https://something.com/myvolume`. Theoretically, this should work for local volumes, as well as any other form of storage representable in pyfileystem

Reply:
> `precomputed://https://something.com/myvolume`

I don't like this because it doesn't look like a valid URI syntax.

As far as I have seen, people use something like `precomputed+https://something.com/myvolume`.

Reply:
Ha.  The plus sign is indeed far better. It even parses correctly with urlparse o.O
Neuroglancer still uses the other 'convention', though (e.g.: `n5://http://example.com/volume.n5`), with the double protocol thing. I'll have a look around, see where it might break, where things might need to be translated back and forth, but I ultimately like it